package com.course.testng;

public class Exception001 {

    public void runTimeExceptionFaile(){

        System.out.println("异常测试");
    }
}
